### PR TITLE
Final changes to info.xml

### PR DESCRIPTION
**Pull request by pmaier1**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Review by jnweiger — APPROVED** [review]
description text polishing is always welcome!

File: appinfo/info.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@ OpenID Connect has become the leading standard for single sign-on and identity p
 - Centralized client management within the Identity Provider
 - Enterprise-grade security through the use of authentication security features (e.g., multi-factor authentication) and policies (e.g., automatic token expiration on certain conditions) provided by Identity Providers
 
-The integration requires an Identity Provider which supports OpenID Connect (e.g., Keycloak, Kopano Konnect, Ping Federate, ADFS or others).
+The integration requires an Identity Provider that supports OpenID Connect (e.g., Keycloak, Kopano Konnect, Ping Federate, ADFS, Azure AD or others).
```
Comment (DeepDiver1975) [inline appinfo/info.xml:18]: wrong - azure ad will come with 1.1.0 - it is not part of 1.0.0

Comment (DeepDiver1975) [reply appinfo/info.xml:18]: @pmaier1 FYI

Comment (jnweiger) [reply appinfo/info.xml:18]: True. azure support is not yet there. But it is an 'e.g.' list. Good enough, I'd say.

Comment (pmaier1) [reply appinfo/info.xml:18]: Yes. This is intended. Thanks for the hint, though :)

Comment (DeepDiver1975) [reply appinfo/info.xml:18]: :shrug:

**Pull request merged**